web/src/main/java/org/akaza/openclinica/control/managestudy/ResolveDiscrepancyServlet.java

```diff
@@ -93,11 +93,21 @@ public Page getPageForForwarding(DiscrepancyNoteBean note, boolean isCompleted)
             }
             // UpdateStudyEvent?event_id=12&ss_id=12
         } else if ("itemdata".equalsIgnoreCase(entityType) || "eventcrf".equalsIgnoreCase(entityType)) {
-            if (currentRole.getRole().equals(Role.MONITOR) || !isCompleted) {
+            /*if (currentRole.getRole().equals(Role.MONITOR) || !isCompleted) {
                  return Page.VIEW_SECTION_DATA_ENTRY_SERVLET;
                 // ViewSectionDataEntry?eventDefinitionCRFId=&ecId=1&tabId=1&studySubjectId=1
             } else {
                 return Page.ADMIN_EDIT_SERVLET;
+            }*/
```

Comment:
if this code is no longer needed, remove it completely. You don't need to comment it out